CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ daq_add_plugin(TPChannelFilter duneDAQModule LINK_LIBRARIES trigger)
 daq_add_plugin(TPSetTee duneDAQModule LINK_LIBRARIES trigger)
 daq_add_plugin(TASetTee duneDAQModule LINK_LIBRARIES trigger)
```

Comment:
should we remove the TASetTee (if we don't expect to use it after this change)?

Reply:
Done